src/components/ApplnBar.js

```diff
@@ -123,9 +144,67 @@ const ApplnBar = () => {
       return <DynamicTabs component1={<ProcessPage />} component2={'c2'} />;
     } else if (page === 'VEHICLE') {
       return <AddVehicle />;
+    } else if (page === 'USER') {
+      return <User />;
+    } else if (page === 'HOME') {
+      return <Home />;
     }
   };
 
+  const onLoginSuccess = (res) => {
+    console.log('Login Success: currentUser:', res.profileObj);
+    setImgUrl(res.profileObj.imageUrl);
+    setAvatarVisible('visible');
+    setLoginVisible(false);
+    setLogoutVisible('visible');
+    refreshTokenSetup(res);
+    setUserName(res.profileObj.name);
+    setEmail(res.profileObj.email);
+    localStorage.setItem('authToken', res.tokenId);
+    refreshTokenSetup(res);
```

Comment:
Do we need this twice?
If yes, can you please add comments around each of them explaining what each of them does.